scripts/extract-svg-html.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const fs = require('fs').promises;
+const { parse } = require('node-html-parser');
+
+module.exports = function extract(opts) {
+  console.log(`[${opts.task}:1/4] Initialise SVG extraction process.`);
+  const iconsPath = path.resolve(__dirname, '..', 'svg');
+  const destCachePath = path.resolve(__dirname, '..', 'src', 'icon-cache.js');
+
+  return new Promise((resolve, reject) => {
+    fs.readdir(iconsPath)
+      .then((icons) => {
+        console.log(`[${opts.task}:2/4] Fetch map of icon file names and SVG data.`);
+        return Promise.all(
+          icons.map((name) => fs.readFile(path.resolve(iconsPath, name), { encoding: 'utf8' }))
+        ).then((xmlContent) => xmlContent.map((xml, i) => [icons[i], parse(xml)]))
+      })
+      .then((svgEntries) => {
+        console.log(`[${opts.task}:3/4] Determine HTML code and viewbox for icon cache.`);
+        svgEntries.map((entry) => [
```

Comment:
We're missing a return here, without which the `build-src` npm script fails

```suggestion
        return svgEntries.map((entry) => [
```